src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4227,9 +4227,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ElementIsExpected" xml:space="preserve">
     <value>element is expected</value>
   </data>
-  <data name="MustCallSetMethodTestData" xml:space="preserve">
-    <value>Must call SetMethodTestData(ConcurrentDictionary) before calling SetMethodTestData(MethodSymbol, ILBuilder)</value>
-  </data>
```

Comment:
The changes around `CompilationTestData` are not specific to this feature but are needed here to allow multiple methods with the same signature.